src/playground/index.jsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
-import { View, Text } from 'react-native';
+import { View, Text, Button } from 'react-native';
```

Comment:
if this is just a playground, then pls do not change it with each pr

Reply:
I think I'll keep it for now as an example for how to navigate to the detail screen